src/app/components/inline-form/index.html

```diff
@@ -151,6 +151,30 @@
 		<header><div class="sky-subsection-heading sky-margin-stacked-default">Edit form</div></header>
 			<p>When users click the Edit button, the view content fades out and the form slides in to replace the view content.</p>
 
+  <header><div class="sky-section-heading">Content</div></header>
+    <header><div class="sky-subsection-heading sky-margin-stacked-default">Primary buttons</div></header>
+  <p>The primary action on the inline form should be Save or Done.</p>
+    <stache-row>
+      <stache-column screenSmall="6">
+        <p>Use Save if the data is saved to the database immediately.</p>
+      </stache-column>
+      <stache-column screenSmall="6">
+        <div class="screenshot screenshot-success">
+          <img src="~/assets/img/guidelines/inline-form/usage-do-block.png" style="width: 100%;" />
+        </div>
+      </stache-column>
+    </stache-row>
+    <stache-row>
+      <stache-column screenSmall="6">
+        <p>Use Done if the inline form is part of a larger form and the data doesn't get saved until the larger form is saved.</p>
```

Comment:
Again, not a big deal, but I'd tweak this to use active voice and rearrange a bit: "Use Done if the inline form does not save data to the database immediately because it is part of a larger form."

Reply:
updated